lib/sidekiq_alive/config.rb

```diff
@@ -33,7 +33,9 @@ def set_defaults
       @queue_prefix = :"sidekiq-alive"
       @server = ENV.fetch("SIDEKIQ_ALIVE_SERVER", "webrick")
       @custom_liveness_probe = proc { true }
-      @shutdown_callback = proc {}
+      @shutdown_callback = proc do
+        Sidekiq::Queue.all.find { |q| q.name == "#{@queue_prefix}-#{SidekiqAlive.hostname}" }&.clear
```

Comment:
This is already done here:
https://github.com/arturictus/sidekiq_alive/blob/master/lib/sidekiq_alive.rb#L78-L90

am I missing something?

Reply:
Will this be a fix to the current implementation?

Reply:
Hey @arturictus! Thanks for the review. Sadly I don't know the codebase very well and Ruby is not my "native" language so I can only say that without this my app left behind tons of old queues. So far with this solution it's not doing that anymore.

Reply:
Hey @gdubicki , could you replace the current 

```ruby
Sidekiq::Queue.new(current_queue).clear
```
here:

https://github.com/arturictus/sidekiq_alive/blob/9d26a9bbe1701726b945e679ae81c39c94cbe0b3/lib/sidekiq_alive.rb#L78-L90

and add your code

```ruby
Sidekiq::Queue.all.find { |q| q.name == "#{@queue_prefix}-#{SidekiqAlive.hostname}" }&.clear
```

As well the callback readme should be undone.

Let me know if you are able to.

Thanks for the PR